app/controllers/custom_labels_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CustomLabelsController < ApplicationController
+  def index
+    render jsonapi: custom_labels.all, include: params[:include]
+  end
+
+  def update
+    data_attributes = JSON.parse(request.body.read)['data']['attributes'] || {}
+    label_id = params[:id].to_i
+    validate(data_attributes, label_id)
+  end
+
+  private
+
+  def custom_labels
+    CustomLabel.configure config
+  end
+
+  def validate(data_attributes, label_id)
```

Comment:
typically we try to follow a format that is roughly:

Reply:
Yep, rubocop was complaining about complexity of the update method which is why I had to split it up the way I did. How do we get around rubocop complaining, do you mean by suppressing its comment about complexity?